README.md

```diff
@@ -54,6 +54,23 @@ For these tasks you may prefer using webpki in combination with libraries like
 Changelog
 =========
 
+* Next release:
+  - *Breaking change*: added `EndEntity::verify_is_valid_tls_client_cert`
+    argument for certificate revocation lists.
+  - Added certificate revocation list (CRL) support.
+  - *Breaking change*: removed deprecated `Time::try_from` fn.
+  - *Breaking change*: removed deprecated `DnsName::From<DnsNameRef<'_>>` trait.
+  - *Breaking change*: replaced `DnsNameRef::AsRef<[u8]>` with
+    `DnsNameRef::AsRef<str>`.
+  - Improved specificity of errors returned from
+    `EndEntityCert::verify_is_valid_tls_client_cert` and
+    `EndEntityCert::verify_is_valid_tls_server_cert`.
+  - Changed `EndEntityCert::verify_is_valid_for_subject_name` to ignore invalid
+    names when verifying cert is valid for a provided subject.
+  - MSRV increased to Rust 1.57.
```

Comment:
I would move the MSRV bump to the bottom, it's probably only relevant to a few users.

Reply:
Sounds good. Changed.